readthedocs/urls.py

```diff
@@ -22,7 +21,6 @@
 
 admin.autodiscover()
 
-handler404 = server_error_404
```

Comment:
Regarding the 404, we are using the Django default on dashboard: https://docs.djangoproject.com/en/2.2/ref/views/#django.views.defaults.page_not_found --It keeps showing our custom `404.html` Maze Found.

Our custom handler is not more required because we are not doing any redirect now, just showing the Maze.

Reply:
I just checked that running the code with DEBUG=False still shows the Maze Found page on any 404 page under the dashboard.